src/app/environments/environments.ts

```diff
@@ -4,7 +4,7 @@
 
 export class environment {
     public static production = false;
-    public static baseUrl = 'https://jbossnewserver.herokuapp.com/';
+    public static baseUrl = 'https://cert-gen-jboss.herokuapp.com/';
```

Comment:
Have you checked this one? Is there working perfectly?

Reply:
Exactly my concern, rest everything seems fine to me too

Reply:
Last, I checked we got `500: Server Error` on user creation

Reply:
``` https://jbossnewserver.herokuapp.com/ ``` was my deploy on heroku, which I deleted.

So I changed it to:
``` https://cert-gen-jboss.herokuapp.com/ ``` (official server url, given in [cert-gen-server repository](https://github.com/JBossOutreach/certificate-generator-server))

That 500: Serer error should be a bug in [certificate-generator-server](https://github.com/JBossOutreach/certificate-generator-server) repository.

Reply:
Yeah, there is some issue with the deployment that we have to check anyway.
Moving to the official URL is right, I think :+1: